packages/@magic-sdk/provider/test/spec/modules/user/getInfo.spec.ts

```diff
@@ -21,3 +21,16 @@ test('Generate JSON RPC request payload with method `magic_get_info` and the act
   expect(requestPayload.method).toBe('magic_get_info');
   expect(requestPayload.params).toEqual([{ walletType: 'metamask' }]);
 });
+
+test('Generate JSON RPC request payload with method `magic_get_info`', async () => {
+  const magic = createMagicSDK();
+  magic.user.request = jest.fn();
+
+  await storage.setItem('mc_active_wallet', 'metamask');
+
+  await magic.user.getInfo();
+
+  const requestPayload = magic.user.request.mock.calls[0][0];
+  expect(requestPayload.method).toBe('magic_get_info');
+  expect(requestPayload.params).toEqual([{ walletType: 'metamask' }]);
+});
```

Comment:
This code is not related to this PR, but it helps this PR to pass the coverage threshold.